Update Runtime 7/Airflow 2.5 release notes (#1597)
astronomer/docs: astro/runtime-release-notes.md

### Airflow 2.5.0

Astro Runtime 7.0.0 includes same-day support for Airflow 2.5.0, which includes a collection of new features, bug fixes, automatic changes, and deprecations. Features include:

- Add comments to task instances and DAG runs in the Airflow UI ([#26457](https://github.com/apache/airflow/pull/26457))
- Clear all task instances in a task group with one click in the Airflow UI ([#26658](https://github.com/apache/airflow/pull/26658)), [#28003](https://github.com/apache/airflow/pull/28003))
- Trigger a task when at least one upstream tasks is successful with new `one_done` trigger rule [#26146](https://github.com/apache/airflow/pull/26146)
- New **Parsed at** metric in the DAG view of the Airflow UI [#27573](https://github.com/apache/airflow/pull/27573)
- Filter datasets in Airflow UI based on recent update events [#26942](https://github.com/apache/airflow/pull/26942)

To learn more, see [What's New in Apache Airflow 2.5](https://www.astronomer.io/blog/whats-new-in-apache-airflow-2-5/) and the [Apache Airflow 2.5.0 release notes](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-5-0-2022-12-02).

### Additional improvements 

> 

Reviewer: cc @chomipi88

> 

Reviewer 2: It would display an action that a user performs in the Airflow UI (eg. pausing a DAG). It would not display updating the deployment as that is outside of Airflow.

### Airflow 2.5.0

> Astro Runtime 7.0.0 includes same-day support for Airflow 2.5.0, which includes a collection of new features, bug fixes, automatic changes, and deprecations. Features include:

Reviewer 3: ```suggestion
Astro Runtime 7.0.0 includes same-day support for Airflow 2.5.0 and the following new features, bug fixes, automatic changes, and deprecations:
```
